mw-webapp/src/component/checkbox/Сheckbox.tsx

```diff
@@ -0,0 +1,83 @@
+import {Indicator, Root} from "@radix-ui/react-checkbox";
+import {CheckIcon} from "@radix-ui/react-icons";
+import clsx from "clsx";
+import styles from "src/component/checkbox/Сheckbox.module.scss";
+
+/**
+ * Checkbox props
+ */
+interface CheckboxProps {
+
+  /**
+   * Custom class for the checkbox.
+   */
+  className?: string;
+
+  /**
+   * Unique identifier for the checkbox.
+   */
+  id: string;
+
+  /**
+   * Content for the label.
+   */
+  label: string;
+
+  /**
+   * The name of the checkbox.
+   */
+  name?: string;
+
+  /**
+   * The value of the checkbox.
+   * @default "on"
+   */
+  value?: string;
+
+  /**
+   * The checked state of the checkbox when it is initially rendered.
+   * @default false
+   */
+  defaultChecked?: boolean;
+
+  /**
+   * When true, prevents the user from interacting with the checkbox.
+   * @default false
+   */
+  disabled?: boolean;
+
+  /**
+   * The required property a checkbox must be checked before submitting a form.
+   * @default false
+   */
+  required?: boolean;
```

Comment:
Pls give an example where we will need checkbox required?

Reply:
Probably an agreement with some privacy terms and etc

Reply:
this prop is not needed in the project at this stage, I removed it